svg.js.d.ts

```diff
@@ -1,3 +1,8 @@
+export = svgjs;
+export as namespace svgjs;
+
+declare var svgjs: svgjs.Library;
+

```

Comment:
could you explain what this code dose?
because when i test the typings with this code it just says that the library dose not have a default export.